extensions/typescript-language-features/src/tsServer/protocol/protocol.d.ts

```diff
@@ -55,6 +56,29 @@ declare module 'typescript/lib/tsserverlibrary' {
 		export interface MapCodeResponse extends Response {
 			body: FileCodeEdits[]
 		}
+		//#endregion
+
+		//#region Paste
+		export interface GetPostPasteImportFixesRequest extends Request {
+			command: 'getPostPasteImportFixes';
+			arguments: GetPostPasteImportFixesRequestArgs;
+		}
+		export type DocumentRange = FileRangeRequestArgs;
+		export type CopyRange = {
```

Comment:
Updated the protocol. I believe we can remove the `CopyRange` and `DocumentRange` here now since `FileSpan` extends `TextSpan`, so we can get the start and end locations from there.

Reply:
I've updated the pull request @mjbvz https://github.com/microsoft/TypeScript/pull/57262